### DISCUSSION TITLE
Como eu crio um —exemplo

### DISCUSSION THREAD

**Discussion by HigorAln** [frontendbr/forum#2169] [Perguntas e Respostas]
Olá pessoal, sou um pouco novo na programação, gostaria de indicações de vídeos ou caso alguém possa explicar como eu posso criar meu próprio —exemplo quando usar o create-next-app para não ter todo o trabalho de colocar tudo em projetos novos

**Comment by ninetails** [comment]
Opa, você diz criar um novo projeto em React utilizando o Create React App utilizando um `--template`? Não sei se entendi...

Seria para criar um app a partir de algo já existente, certo?

Tem uma parte da documentação que diz sobre [custom templates (em inglês)](https://create-react-app.dev/docs/custom-templates/) e inclusive te dá 2 repos como exemplo.

Se for isso, também tem como criar projetos a partir de [templates utilizando o próprio GitHub](https://docs.github.com/pt/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/creating-a-repository-from-a-template).

Também existem outras ferramentas de _scaffolding_, como o [Yeoman](https://yeoman.io/).

Se não for isso, explica melhor pra gente o que seria esse `--exemplo` :D

**Reply by HigorAln** [reply]
e isso mesmo ^^ não soube explicar direito kk, mas e isso mesmo, muito obrigado!!!
